src/test/groovy/com/stormpath/tck/login/LoginIT.groovy

```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.stormpath.tck.login
```

Comment:
There's an assumed package for each class in Groovy (based on the folder structure), and I noticed a warning. So I removed the package declaration.